app/assets/javascripts/dashboard/notes.js.coffee

```diff
@@ -42,4 +42,15 @@ $ ->
       type: 'GET'
       url: '/dashboard/notes/new'
       data: data
+
+  $(document).on 'click', 'button.notes.cancel',  ->
```

Comment:
Talked to Stuart about this, this does the same thing as the notes list button handler, should they be combined? Potentially we might see different functionality in the future for cancel, but right now they're the same